xcodeproj/defs.bzl

```diff
@@ -39,3 +40,6 @@ xcode_provisioning_profile = _xcode_provisioning_profile
 # Re-exporting APIs
 xcode_schemes = _xcode_schemes
 xcschemes = _xcschemes
+
+# Re-exporting aspects
```

Comment:
Can we just have it be a top-level file? I don’t want to add new symbols to this bzl file.